### PR TITLE
RIOT: update to 2020.07

### PR DESCRIPTION
**Pull request by miri64**
Check all tasks if the READMEs still work.

### CODE REVIEW

File: task-06/README.md [hunk 1]
```diff
@@ -26,29 +26,32 @@ USEMODULE += gnrc_sock_udp
    udp                  send udp packets
    udps                 start udp server
    reboot               Reboot the node
+   version              Prints current RIOT_VERSION
+   pm                   interact with layered PM subsystem
    ps                   Prints information about running threads.
    ping6                Ping via ICMPv6
+   ping                 Ping via ICMPv6
```
Comment (cgundogan) [inline task-06/README.md:33]: hmm .. I am seeing this just now. Having two times the same help message looks strange to me. Something alone the lines of `see ping6` or `behaves like ping6` would be IMO better than duplicating the help text. It's not part of this PR, though.

**Review by cgundogan — APPROVED** [review]
ACK

File: task-06/README.md (hunk 1 above)
Comment (miri64) [reply task-06/README.md:33]: You are welcome to change that. I'm not sure however that is the correct help text for that alias. `Alias for ping6` maybe?
